css/card.css

```diff
@@ -14,7 +14,63 @@
   box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
   -webkit-box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
 }
-
+.card-d {
+  top:30px;
+  transform: rotate(30deg);
+  width: 75px;
+  float: left;
+  height: 100px;
+  background: #fff;
+  position: relative;
+  margin-right: 10px;
+  margin-left: 10%;
+  border-radius: 10px;
+  box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
+  -webkit-box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
+  z-index: auto;
+}
+.card-d1 {
+  transform: rotate(45deg);
+  width: 75px;
+  float: left;
+  height: 100px;
+  background: #fff;
+  position: relative;
+  margin-right: 10px;
+  margin-left: -35px;
+  border-radius: 10px;
+  box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
+  -webkit-box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
+  z-index: 1;
+}
+.card-d2 {
+  transform: rotate(60deg);
+  width: 75px;
+  float: left;
+  height: 100px;
+  background: #fff;
+  position: relative;
+  margin-right: 10px;
+  margin-left: -35px;
+  border-radius: 10px;
+  box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
+  -webkit-box-shadow: 3px 3px 7px rgba(0,0,0,0.3);
+  z-index: 2;
+}
+.card-d3 {
```

Comment:
Baigais copy/paste šīm te 3 klasēm, vajadzētu padomāt/palasīt, kā var to daudz īsāk uzrakstīt!

Reply:
Piekrītu, tagad pie tā strādāju!